### PR TITLE
pass secrets to Terraform through Concourse to Terraform variables

### PR DESCRIPTION
**Pull request by dan-hill2802**
Signed-off-by: danhill <danhill@digital.uc.dwp.gov.uk>

### CODE REVIEW

**Review by iw-gtadam — COMMENTED** [review]
LGTM

**Review by jimbarker — APPROVED** [review]

File: ci/jobs/pull_request.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ jobs:
   - .: (( inject meta.plan.terraform-bootstrap ))
     input_mapping:
       dataworks-metrics-infrastructure: dataworks-metrics-infrastructure-pr
+    params:
+      TF_WORKSPACE: management
+      AWS_ROLE_MGT_DEV: arn:aws:iam::((aws_account.management)):role/ci
```
Comment (jimbarker) [inline ci/jobs/pull_request.yml:17]: Should this be AWS_ROLE_MGT instead of AWS_ROLE_MGT_DEV?

File: ci/jobs/management.yml [hunk 2]
```diff
@@ -13,14 +13,19 @@ jobs:
       - .: (( inject meta.plan.terraform-bootstrap ))
         params:
           TF_WORKSPACE: management
+          AWS_ROLE_MGT_DEV: arn:aws:iam::((aws_account.management)):role/ci
```
Comment (jimbarker) [inline ci/jobs/management.yml:16]: Should this be AWS_ROLE_MGT instead of AWS_ROLE_MGT_DEV?

**Review by jimbarker — APPROVED** [review]

File: ci/jobs/management.yml (hunk 2 above)
Comment (dan-hill2802) [reply ci/jobs/management.yml:16]: it's correct in terms of matching, but the name isn't good, let me rename it

**Review by jimbarker — CHANGES_REQUESTED** [review]
Just need to check a couple of references

**Pull request merged**